README.md

```diff
@@ -52,13 +52,18 @@ const apiKeyName = "Copy your API Key name here.";
 
 const apiKeyPrivateKey = "Copy your API Key's private key here.";
 
-const coinbase = new Coinbase(apiKeyName, apiKeyPrivateKey);
+const coinbase = new Coinbase({ apiKeyName: apiKeyName, privateKey: apiKeyPrivateKey });
+```
+
+If you are using a CDP Server-Signer to manage your private keys, enable it with the constuctor option:
+```typescript
+const coinbase = new Coinbase({ apiKeyName: apiKeyName, privateKey: apiKeyPrivateKey, useServerSigner: true })
 ```
 
 Another way to initialize the SDK is by sourcing the API key from the json file that contains your API key, downloaded from CDP portal.
 
 ```typescript
-const coinbase = Coinbase.configureFromJson("path/to/your/api-key.json");
+const coinbase = Coinbase.configureFromJson({ filePath: "path/to/your/api-key.json" });
```

Comment:
we should fix this in docs as well if this is the case @erdimaden

Reply:
agreed, we need to update the docs.

Reply:
+1 docs need to be updated to use new paradigm

Reply:
@John-peterson-coinbase let's use `privateKey` instead of `apiKeyPrivateKey` so we can just use 
```suggestion
const coinbase = new Coinbase({ apiKeyName, privateKey });
```